docs/content/error/$compile/noctrl.ngdoc

```diff
@@ -0,0 +1,68 @@
+@ngdoc error
+@name $compile:noctrl
+@fullName Controller is required.
```

Comment:
Should this full name be more like: `"controller as" identifier required to bind to controller`?